RefactorThis.API/Controllers/ProductsController.cs

```diff
@@ -38,16 +38,16 @@ public async Task<IActionResult> Get(Guid id, CancellationToken cancellationToke
         [HttpPost]
         public async Task<IActionResult> Post(CreateProductCommand command, CancellationToken cancellationToken)
         {
-            var response = await mediator.Send(command, cancellationToken);
+           await mediator.Send(command, cancellationToken);
 
-            return Ok(response);
+            return Ok();
         }
 
         [HttpPut("{id}")]
         public async Task<IActionResult> Update(UpdateProductCommand command, CancellationToken cancellationToken)
         {
-            var response = await mediator.Send(command, cancellationToken);
-            return Ok(response);
+            await mediator.Send(command, cancellationToken);
+            return Ok();
```

Comment:
// now I wish to return a 204 no content response to the user 
    // instead of the 200 OK response
    `return new HttpStatusCodeResult(HttpStatusCode.NoContent);`

Reply:
`return NoContent();`